hud_api_replace/geocode.py

```diff
@@ -25,18 +34,19 @@ def _extract_zip_code(key):
 
 def geocode_get_data(address, zip_only=False):
     """Main function to obtain geocoding information."""
-    address = str(address) + ", usa"
     try:
         cached = CachedGeodata.objects.all().filter(expires__gte=time.time()).get(key=address)
     except ObjectDoesNotExist:
         if zip_only:
-            mb_result = Mapbox(address ,types='postcode')
+            mb_result = PermanentMapbox(address,
+                                        types='postcode',
+                                        country='us')
```

Comment:
There have been, I don't know about today. Also, there's #20 -- we shouldn't *just* be coding zip codes